### PR TITLE
97 user management gui

### PR DESCRIPTION
**Pull request by jstucke**
Addresses issue #97

### CODE REVIEW

File: src/helperFunctions/web_interface.py [hunk 1]
```diff
@@ -82,6 +82,8 @@ def get_template_as_string(view_name):
 
 
 def _auth_is_disabled(user):
+    if isinstance(user, UserMixin):
+        return isinstance(user, AnonymousUser)
```
Comment (jstucke) [inline src/helperFunctions/web_interface.py]: Better test for proxy object and in this case use "_get_current_object()"

File: src/web_interface/components/component_base.py [hunk 2]
```diff
@@ -4,10 +4,12 @@
 
 
 class ComponentBase(metaclass=ABCMeta):
-    def __init__(self, app, config, api=None):
+    def __init__(self, app, config, api=None, user_db=None, user_db_interface=None):
```
Comment (jstucke) [inline src/web_interface/components/component_base.py]: move to __init__ of concrete component

File: src/web_interface/templates/base.html [hunk 3]
```diff
@@ -100,7 +101,9 @@
                 <div class="navbar-brand hidden-sm hidden-md" style="vertical-align: middle; margin-left: 15px">
                     <span><img height="25px" src="{{ url_for('static', filename='fact_logo_inv.svg') }}"></span>
                 </div>
-                <div class="navbar-brand hidden-sm hidden-md" style="vertical-align: middle">
+                <div class="navbar-brand hidden-sm hidden-md v
```
Comment (jstucke) [inline src/web_interface/templates/base.html]: ?

File: src/web_interface/templates/user_management/edit_user.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                    {{ user.active }}
+                </td>
+            </tr>
+
+            <!-- user roles -->
+            <tr>
+                <td class="active result" style="text-align: right; vertical-align: middle;">
+                    Roles:
+                </td>
+                <td class="active result" style="padding-right: 90px">
```
Comment (weidenba) [inline src/web_interface/templates/user_management/edit_user.html]: Is it possible to add tool tips indicating the privileges of a role? That would be nice.

**Pull request merged**